Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorInstance.cpp

```diff
@@ -558,7 +557,7 @@ namespace AZ
 
                 // Update the morph weights for every lod. This does not mean they will all be dispatched, but they will all have up to date weights
                 // TODO: once culling is hooked up such that EMotionFX and Atom are always in sync about which lod to update, only update the currently visible lods [ATOM-13564]
-                for (uint32_t lodIndex = 0; lodIndex < m_actorInstance->GetActor()->GetNumLODLevels(); ++lodIndex)
+                for (uint32_t lodIndex = 0; lodIndex < aznumeric_cast<uint32_t>(m_actorInstance->GetActor()->GetNumLODLevels()); ++lodIndex)
```

Comment:
perhaps have the numeric cast done outside the loop

Reply:
Fixed.